src/OpenTelemetry.Api/Context/RuntimeContext.cs

```diff
@@ -39,35 +39,46 @@ public static class RuntimeContext
         /// <summary>
         /// Register a named context slot.
         /// </summary>
-        /// <param name="name">The name of the context slot.</param>
+        /// <param name="slotName">The name of the context slot.</param>

```

Comment:
I don't really know if this is idiomatic to C# or general OOP.
Let's say we have something like this:

```csharp
class Horse
{
    private string name;
    private int age;
    public Person(string name, int age)
    {
    }
}
```

or this:

```csharp
class Horse
{
    private string horseName;
    private int horseAge;
    public Horse(string horseName, int horseAge)
    {
    }
}
```